en/developer_guide/routing.rst

```diff
@@ -102,27 +114,30 @@ Declaring differents loaders as services in ``routing.yml`` is mandatory to gene
         resource: '.'
         type: orchestra_redirection
 
-See `Symfony Loader documentation`_ for more information.
+You also have to modify the project configuration, in the ``config.yml`` file :
+
+.. code-block:: yaml
+
+    open_orchestra_front:
+        routing_type: file
+
+How to choose between both of the solution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this part, I would not tell you which routing type you should use, but only list a some condition to use
+one or the other.
 
-URL Generator
-~~~~~~~~~~~~~
+You are not using the Open Orchestra Backoffice :
 
-``OpenOrchestraUrlGenerator`` generates URLs from routes.
-The URL Generator is overridden to take `RequestStack`_ in argument and get the ``aliasId`` from it.
-It will be used to adapt this route name with route names generated by Open Orchestra ``Route Loader``.
+* Use the file routing, or you will have to manually update the ``route_document`` collection
 
-Generator dumper
-~~~~~~~~~~~~~~~~
+You have a lot of pages, and they are updated by a big number of people all the time :
 
-``OpenOrchestraGeneratorDumper`` extends ``PhpGeneratorDumper``, it will dump the routing in the Symfony cache
-file to avoid any request to the database.
+* Use the database routing, no cache file will be updated each time a route is created
 
-Router
-~~~~~~
+There is only a few update of the pages :
 
-``OpenOrchestraRouter`` extends ``Symfony\Bundle\FrameworkBundle\Routing\Router`` to add the ``RequestStack``
-in the OpenOrchestraUrlGenerator constructor.
+* If you want to lower the number of query to the database, use the file routing
+* Use the database routing if there is no restriction
```

Comment:
After a new read, I change my note: I think the first point should be removed as I don't find it honnest. Or at least it should be reexplained and placed on the third position: points 2 and 3 are objectives helps relying on performance issues. They are top level advices.
But point 1 is only based on "you don't need to code anything new". Using that router with frequent nodes update will lead to bad performances and it should mentionned. So I'll add something like: "can be used without coding missing parts due to missing BO (node routes update) but can lead to bad performances if there's a lot of updates"